packages/lib-classifier/src/store/ClassificationStore.js

```diff
@@ -132,10 +132,12 @@ const ClassificationStore = types
 
     function completeClassification () {
       const classification = self.active
+      const subjectDimensions = getRoot(self).subjectViewer.dimensions.toJSON()
```

Comment:
Could we use `toJS` instead of `toJSON`? `toJS` gives us options about how to convert maps, if the data structure is using one.

Reply:
Sure.